lib/pseudolocalization.rb

```diff
@@ -23,8 +26,28 @@ def respond_to_missing?(name, include_private = false)
       end
 
       def translate(locale, key, options)
+        return original_backend.translate(locale, key, options) if key_ignored?(key)
+
         ::Pseudolocalization::I18n::Pseudolocalizer.pseudolocalize(original_backend.translate(locale, key, options))
       end
+
+      private
+
+      def key_ignored?(key)
+        return false unless ignores
+
+        ignores.any? do |ignore|
+          case ignore
+          when Regexp
+            key.to_s.match(ignore)
+          when String
+            File.fnmatch(ignore, key.to_s)
+          else
+            Rails.logger.tagged('Pseudolocalization I18n').error 'Ignore type unsupported. Expects an array of (mixed) Regexp or Strings.'
+            return false
```

Comment:
```suggestion
           false
```

I think we can remove the `return` here.

By the way, good idea explicitely `false`-ing here. Turns out Rails logger returns truthy.
<img width="952" alt="Screen Shot 2021-02-03 at 14 23 30" src="https://user-images.githubusercontent.com/1557529/106702332-866f4c80-662b-11eb-90dd-ea47d3703a04.png">

Reply:
Fixed